src/sentry/static/sentry/app/views/admin/adminWarnings.tsx

```diff
@@ -3,13 +3,16 @@ import React from 'react';
 import AsyncView from 'app/views/asyncView';
 import {t} from 'app/locale';
 
-export default class AdminSettings extends AsyncView {
-  getEndpoints() {
-    return [['data', this.getEndpoint()]];
-  }
+type Data = {
+  groups: [groupName: string, grouppedWarnings: string[]][];
+  warnings: string[];
+};
+
+type State = AsyncView['state'] & {data: Data};
 
-  getEndpoint() {
-    return '/internal/warnings/';
+export default class AdminSettings extends AsyncView<{}, State> {
```

Comment:
Shouldn't this be named `AdminWarnings`, doesn't affect the code but it feels wrong

Reply:
Yes it should lol